Sources/Automerge/Document.swift

```diff
@@ -582,10 +582,34 @@ public final class Document: @unchecked Sendable {
 
     /// Establish a cursor at the position you specify in the list or text object you provide.
     ///
+    /// In collaborative applications, maintaining stable cursor positions is crucial. Traditional index-based
+    /// positions can become outdated due to document modifications. This method ensures the cursor stays
```

Comment:
```suggestion
    /// positions become outdated when you or collaborators modify the document. This method provides a stable reference to a character that stays
```

Reply:
Indeed, your recent grammar tweaks have rendered it redundant, so we can remove it for clarity.  Addressed in: f7657f61896f70ca1d6dc36b74355274daf20b8a